dhis-2/dhis-services/dhis-service-reporting/src/test/java/org/hisp/dhis/interpretation/InterpretationServiceTest.java

```diff
@@ -252,7 +312,8 @@ public void testLikeInterpretation()
     }
 
```

Comment:
Can we add some additional tests?

Reply:
We added some more tests for mentioning. If you have anything in particular you would like us to further testing please let us know...

Reply:
Thank you.